### DISCUSSION TITLE
2024/quotes/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/shane0.github.io#58] [General]
# 2024/quotes/

https://shanenull.com/2024/quotes/

**Comment by shane0** [comment]
# quotes

- updating click workflow/ app to include quotes
- design based on quotes
- https://quotefancy.com/quote/964386/Cormac-McCarthy-Where-in-this-pukehole-can-a-man-get-a-drink-he-said
- linus feynman cormac cartesian stoic etc this goes way back need to find my original repos
- https://shanenull.com/stoicism/mentors/

**Reply by shane0** [reply]
https://rosettacode.org/wiki/Rosetta_Code

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2023/collection/

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2021/favorites/

**Reply by shane0** [reply]
trekcho - https://github.com/shane0/buddhism/discussions/82#discussioncomment-8575393

**Reply by shane0** [reply]
https://shanenull.com/stoicism/entropy/
